### ISSUE TITLE
TypeError on single issue template

### ISSUE THREAD

**Issue by chriswallace** [LiftUX/IssuePress#101]
Whenever I try to view a single issue, the content does not load and I see this error:

http://cl.ly/image/1G0Y3p2e3q29

```
TypeError: undefined is not a function
    at new <anonymous> (http://issuepress.co/content/plugins/issuepress/build/main.js?ver=0.0.1:2387:12)
    at d (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:30:452)
    at Object.instantiate (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:31:80)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:62:33
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:49:43
    at q (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:7:386)
    at H (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:48:406)
    at f (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:42:399)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular.min.js?ver=3.9.1:42:67
    at link (https://ajax.googleapis.com/ajax/libs/angularjs/1.2.12/angular-route.min.js?ver=3.9.1:7:348) <div class="left-column ng-scope" data-ng-view=""> 
```

---

IssuePress Data:

username: chriswallace
email: chris@liftux.com
firstname: Chris
lastname: Wallace
display_name: Chris Wallace
ID: 1
gravatar_id: de5e8c95f06ac55e9b297c7c3e1de33b

Sent via [IssuePress](http://issuepress.co)

**Comment by matthewsimo** [comment]
Update your version and take a look now, should be resolved in 7261b30e3fdb92d8a05311365258f79e54c5c7fb

**Comment by chriswallace** [comment]
Closing out, fixed.
